recipes/pblaa/2.4.2/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "pblaa" %}
+{% set version = "2.4.2" %}
+{% set laa_sha256 = "563a2f68c370f9e4f6eb3ac6656ee8a37be3abdae64124d7d077488dcb332172" %}
+{% set laagc_sha256 = "ec366dcbf1c6197e9baf4cb5147c8b41d5d76d860ed835ae6407c88d870205e0" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/PacificBiosciences/pblaa/releases/download/v{{ version }}/laa # [linux]
+    sha256: {{ laa_sha256 }} # [linux]
+  - url: https://github.com/PacificBiosciences/pblaa/releases/download/v{{ version }}/laagc # [linux]
+    sha256: {{ laagc_sha256 }} # [linux]
+
+about:
+  home: https://github.com/ylipacbio/pblaa
+  license: Pacific Biosciences Proprietary
+  summary: PacBio tool to deconvolute mixtures of alleles and loci into phased consensus sequences.
+
+extra:
+  recipe-maintainers:
+    - ylipacbio
+    - armintoepfer
+    - pb-dseifert
+    - mhsieh
+
+build:
+  number: 0
+  skip: True # [osx]
+
```

Comment:
spurious empty line

Reply:
Thanks for the review, I have removed spurious empty lines from this PR and squashed the commits.